bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -210,7 +215,7 @@ protected void initModel() {
                     smartImages[index] = baseResourcePath + DOT +
                         selector + DOT + jpegQuality + DOT + width + DOT + extension +
                         (inTemplate ? Text.escapePath(templateRelativePath) : "") +
-                        (lastModifiedDate > 0 ? "/" + lastModifiedDate + DOT + extension : "");
+                        (lastModifiedDate > 0 ? "/" + lastModifiedDate + (StringUtils.isNotBlank(imageName)? "/" +imageName :"") + DOT + extension : "");
```

Comment:
Formatting, can we space the ternary operators:
`(StringUtils.isNotBlank(imageName) ? "/" +imageName : "")`